src/libs/OptionsListUtils.js

```diff
@@ -1317,7 +1318,24 @@ function getOptions(
     const searchValue = parsedPhoneNumber.possible ? parsedPhoneNumber.number.e164 : searchInputValue.toLowerCase();
 
     // Filter out all the reports that shouldn't be displayed
-    const filteredReports = _.filter(reports, (report) => ReportUtils.shouldReportBeInOptionList(report, Navigation.getTopmostReportId(), false, betas, policies));
+    const filteredReports = _.filter(reports, (report) => {
+        const {parentReportID, parentReportActionID} = report || {};
+        const canGetParentReport = parentReportID && parentReportActionID && allReportActions;
+
+        return ReportUtils.shouldReportBeInOptionList({
+            report,
+            currentReportId: Navigation.getTopmostReportId(),
+            betas,
+            policies,
+            doesReportTransactionThreadHaveViolations:
+                betas.includes(CONST.BETAS.VIOLATIONS) &&
+                ReportUtils.doesTransactionThreadHaveViolations({
+                    report,
+                    transactionViolations,
+                    parentReportAction: canGetParentReport ? lodashGet(allReportActions, [parentReportID, parentReportActionID], {}) : {},
+                }),
```

Comment:
Pull this into a variable to make this easier to read

Reply:
done